frappe/www/contact.py

```diff
@@ -53,7 +53,9 @@ def send_message(subject="Website Query", message="", sender=""):
 	# send email
 	forward_to_email = frappe.db.get_single_value("Contact Us Settings", "forward_to_email")
 	if forward_to_email:
-		frappe.sendmail(recipients=forward_to_email, sender=sender, content=message, subject=subject)
+		frappe.sendmail(
+			recipients=sender, reply_to=sender, bcc=forward_to_email, content=message, subject=subject
+		)
```

Comment:
it kinda doesn't make sense to keep the "sender" (who in this case is the person who's filled out the form) as the recipient.

we could just remove the `sender` param and just keep them in `reply_to`? the default outgoing email account will get picked up by default

the recipient can be reverted back to `forward_to_email`?

Reply:
It makes perfect sense, because the person who filled out the form will want a copy to remind them that they filled it out.

Reply:
If we want to send the person who's filled the form some sort of "copy" ..we should send an email like "Thank you for reaching out. We will try to reply back.." or something along those lines rather than just sending their query back to them :laughing: